cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java

```diff
@@ -66,6 +66,10 @@
             converter = LocalizedAssetBodyInheritanceMode.class)
     LocalizedAssetBody.InheritanceMode inheritanceMode = LocalizedAssetBody.InheritanceMode.USE_PARENT;
 
+    @Parameter(names = {"--status"}, required = false, description = "To choose the translations used to generate the file based on their status. APPROVED: only includes translations that are approved. APPROVED_OR_NEEDS_REVIEW: includes translations that are accepted or that need review. ALL: includes all translations available even if they need re-translation (\"rejected\" translations are always excluded as they are considered harmful).",
```

Comment:
Oh actually wait you've put "approved".

It should be "ACCEPTED" here not "APPROVED" so that it matches statuses shown in the workbench/for the end user. ("approved" is used in the backend)